Documentation/Blazorise.Docs/Blazorise.Docs.csproj

```diff
@@ -53,10 +53,6 @@
   <ItemGroup>
     <None Include="compilerconfig.json" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Generally the web apps have the resources, but in this case, this is a component library for a demo, so the reason why I had images allocated in this project. Is because this is the project that theoritically should know about it's own assets. Otherwise you have to always manually add the images to web projects that consume this library.
However since this is for our internal use and I'm guessing the parameters source binding looks better with the regular relative path, I guess this is better...

Reply:
We will not have more than one web project that serves documentation, so it is better to have them there. And in case we do have more projects it is an easy change.